tasks/manager/src/test/java/org/infinispan/tasks/TaskManagerTest.java

```diff
@@ -58,7 +58,7 @@ public void testStoredEngines() {
 
    public void testRunTask() throws InterruptedException, ExecutionException {
       memoryLogger.reset();
-      CompletableFuture<String> okTask = taskManager.runTask(DummyTaskTypes.SUCCESSFUL_TASK.name(), new TaskContext().logEvent(true));
+      CompletableFuture<String> okTask = taskManager.<String>runTask(DummyTaskTypes.SUCCESSFUL_TASK.name(), new TaskContext().logEvent(true)).toCompletableFuture();
```

Comment:
I'd rather declare the variable as `CompletionStage` here and use `CompletionStages.join()` on the next line.

Reply:
Sure